client/src/App.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div>
+    <todo-list v-bind:todos="todos"></todo-list>
+    <create-todo v-on:create-todo="addTodo"></create-todo>
+  </div>
+</template>
+
+<script>
+import TodoList from './components/TodoList';
+import CreateTodo from './components/CreateTodo';
+
+const url = 'https://localhost:9393/';
+
+export default {
+  components: {
+    TodoList,
+    CreateTodo,
+  },
+  // data function avails data to the template
+  data() {
+    return {
+      todos: [],
+    };
+  },
+  methods: {
+    addTodo(todo) {
+      this.todos.push({
+        title: todo.title,
+        project: todo.project,
+        done: todo.done,
+      });
+    },
+    getTodos() {
+      axios.post(url + '/todos')
```

Comment:
You probably want to `GET /todos`

Reply:
I was still implementing this method. I'll remove it to make a clean PR and work on it later